src/main/java/no/sikt/sws/PreFixStripper.java

```diff
@@ -0,0 +1,88 @@
+package no.sikt.sws;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import no.sikt.sws.constants.ApplicationConstants;
+import no.sikt.sws.models.gateway.Builder;
+import no.sikt.sws.models.opensearch.OpenSearchCommand;
+import no.unit.nva.commons.json.JsonUtils;
+import nva.commons.apigateway.exceptions.BadRequestException;
+import nva.commons.core.JacocoGenerated;
+
+import static no.sikt.sws.constants.ApplicationConstants.EMPTY_STRING;
+import static no.sikt.sws.constants.ApplicationConstants.REQUIRED_PARAMETER_IS_NULL;
+
+@JacocoGenerated
+public class PreFixStripper {
```

Comment:
You added another file called Prefixer, and this one with camelcase PreFixStripper. Please choose one: PreFix or Prefix